packaging/linux/Arch/PKGBUILD

```
@@ -69,43 +114,74 @@ prepare() {
 build() {
     export BRANCH="@GITHUB_BRANCH@"
     export BUILD_VERSION="@BUILD_VERSION@"
-    export COMMIT="@GITHUB_COMMIT@"
+    export COMMIT="${_commit}"
 
     export CC="gcc-${_gcc_version}"
     export CXX="g++-${_gcc_version}"
 
     export CFLAGS="${CFLAGS/-Werror=format-security/}"
     export CXXFLAGS="${CXXFLAGS/-Werror=format-security/}"
 
-    cmake \
-        -S "$pkgname" \
-        -B build \
-        -Wno-dev \
-        -D BUILD_DOCS=OFF \
-        -D BUILD_WERROR=ON \
-        -D CMAKE_INSTALL_PREFIX=/usr \
-        -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine \
-        -D SUNSHINE_ASSETS_DIR="share/sunshine" \
-        -D SUNSHINE_PUBLISHER_NAME='LizardByte' \
-        -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev' \
-        -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+    export CUDA_PATH=/opt/cuda
+    export NVCC_CCBIN="/usr/bin/g++-$_cuda_gcc_version"
+
+    export MAKEFLAGS="-j$(nproc)"
+
+    local _cmake_options=(
+      -S "$pkgname"
+      -B build
+      -Wno-dev
+      -D BUILD_DOCS=OFF
+      -D BUILD_WERROR=ON
+      -D CMAKE_INSTALL_PREFIX=/usr
+      -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine
+      -D SUNSHINE_ASSETS_DIR="share/sunshine"
+      -D SUNSHINE_PUBLISHER_NAME='LizardByte'
+      -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev'
+      -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+      -D SUNSHINE_ENABLE_CUDA=ON
+      -D SUNSHINE_ENABLE_DRM=ON
+      -D SUNSHINE_ENABLE_TRAY=ON
+      -D SUNSHINE_ENABLE_VAAPI=ON
+      -D SUNSHINE_ENABLE_WAYLAND=ON
+      -D SUNSHINE_ENABLE_X11=ON
+    )
+
+    if [[ "${_use_cuda::1}" == "t" ]]; then
+      _cmake_options+=(-DCUDA_FAIL_ON_MISSING=ON)
+    else
+      _cmake_options+=(-DCUDA_FAIL_ON_MISSING=OFF)
+    fi
+
+    if [[ "${_run_unit_tests::1}" == "t" ]]; then
+      _cmake_options+=(-DBUILD_TESTS=ON)
+    else
+      _cmake_options+=(-DBUILD_TESTS=OFF)
+    fi
```

```suggestion
    if [[ "${_use_cuda::1}" != "t" ]]; then
      _cmake_options+=(-DCUDA_FAIL_ON_MISSING=OFF)
    fi

    if [[ "${_run_unit_tests::1}" != "t" ]]; then
      _cmake_options+=(-DBUILD_TESTS=OFF)
    fi
```

We could just handle the non default case.

---

Done